service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -105,29 +117,36 @@ public OrderTypeAndDocument buildOrderTypeAndDocument(OrderTypeAndDocument typeA
         switch (caseData.getOrderTypeAndDocument().getType()) {
             case BLANK_ORDER:
                 orderTemplateBuilder
-                    .put("orderType", BLANK_ORDER)
                     .put("orderTitle", defaultIfNull(caseData.getOrder().getTitle(), "Order"))
                     .put("childrenAct", "Children Act 1989")
                     .put("orderDetails", caseData.getOrder().getDetails());
                 break;
             case CARE_ORDER:
                 orderTemplateBuilder
-                    .put("orderType", CARE_ORDER)
                     .put("orderTitle", "Care order")
                     .put("childrenAct", "Section 31 Children Act 1989")
                     .put("orderDetails", careOrderDetails(getChildrenDetails(caseData).size(),
                         caseData.getCaseLocalAuthority()));
                 break;
+            case SUPERVISION_ORDER:
+                orderTemplateBuilder
+                    .put("orderTitle", "Supervision order")
```

Comment:
minor: maybe GeneratedOrderType can expose label getter, which can be used here to produce title